index.js

```diff
@@ -76,6 +76,14 @@ module.exports = function Expirer(timeoutMs, db, checkIntervalMs) {
 
 	expirer.touch = expirer.emit.bind(expirer, 'touch')
 	expirer.forget = expirer.emit.bind(expirer, 'forget')
+	expirer.createIfNotExists = function createIfNotExists(key) {
+		db.get(key, function (err, value) {
+			if (err && err.notFound) {
+				console.log('creating ' + key)
```

Comment:
Whoops, fixed in https://github.com/TehShrike/expire-unused-keys/pull/9/commits/4ff5c2457dfdfcd5f493367303624307c6da0f87.